	For a cross compilation Unix => mingw we certainly can't have this.
	This is just a minimal fix to build static libraries, but certainly
	not sufficient for icu-config.

	The whole scheme is broken for cross compilation, where
	CURR_FULL_DIR and CURR_SRCCODE_FULL_DIR should come from
	the native configuration of the build system.

diff -ur icu-4.4.orig/source/config/mh-mingw icu-4.4/source/config/mh-mingw
--- icu-4.4.orig/source/config/mh-mingw	2010-03-16 06:14:02.000000000 +0100
+++ icu-4.4/source/config/mh-mingw	2010-03-26 20:15:09.208611976 +0100
@@ -72,10 +72,10 @@
 # The #M# is used to delete lines for icu-config
 # Current full path directory.
 #CURR_FULL_DIR=$(shell pwd -W)#M# for MSYS
-CURR_FULL_DIR=$(subst \,/,$(shell cmd /c cd))#M# for Cygwin shell
+#CURR_FULL_DIR=$(subst \,/,$(shell cmd /c cd))#M# for Cygwin shell
 # Current full path directory for use in source code in a -D compiler option.
 #CURR_SRCCODE_FULL_DIR=$(subst /,\\\\,$(shell pwd -W))#M# for MSYS
-CURR_SRCCODE_FULL_DIR=$(subst \,/,$(shell cmd /c cd))#M# for Cygwin shell
+#CURR_SRCCODE_FULL_DIR=$(subst \,/,$(shell cmd /c cd))#M# for Cygwin shell
 
 ## Compilation rules
 %.$(STATIC_O): $(srcdir)/%.c
